README.md

```diff
@@ -25,6 +25,15 @@ How to cite Autosubmit:
     2016 International Conference on High Performance Computing & Simulation (HPCS), 
     Innsbruck, 2016, pp. 895-900. doi: 10.1109/HPCSim.2016.7568429
 
+AUTOSUBMIT IS MAINTAINED ON THE FOLLOWING MACHINES
+==================================================
+
+- bscesautosubmit{01/02}.bsc.es (Autosubmit 3, BSC machine)
+- bsceshub02.bsc.es (Autosubmit 3 and Autosubmit 4, BSC machine)
+- AEMET (Autosubmit 3 and Autosubmit 4, BSC machine)
+- climate-dt (Autosubmit 4, CSC machine)
+- EDITO-infra (Autosubmit 4, MOI and VLIZ infrastructure)
```

Comment:
Maybe this:

- bscesautosubmit{01/02}.bsc.es -- Autosubmit 3, BSC virtual machine
- bsceshub{02/03/04}.bsc.es -- Autosubmit 3 and Autosubmit 4, BSC virtual machine
- climate-dt -- Autosubmit 4, CSC virtual machine
- EDITO-infra -- Autosubmit 4, MOi and VLIZ cloud infrastructure (Docker, Kubernetes)
- AEMET -- Autosubmit 3 and Autosubmit 4, AEMET virtual machine